src/Common/src/System/Security/Cryptography/Asn1/asn.xsd

```diff
@@ -86,6 +86,7 @@
               <xs:enumeration value="byte" />
               <xs:enumeration value="int" />
               <xs:enumeration value="BigInteger" />
+              <xs:enumeration value="ReadOnlyMemory&lt;byte&gt;" />
```

Comment:
Thinking about it more, I'd probably just call this "ReadOnlyMemory" (leaving the "\<byte\>" as implied) to avoid the &amp;lt; problem.

Reply:
Ok, will change it.